sql-migrate/config.go

```diff
@@ -32,11 +33,24 @@ func ConfigFlags(f *flag.FlagSet) {
 }
 
 type Environment struct {
-	Dialect    string `yaml:"dialect"`
-	DataSource string `yaml:"datasource"`
-	Dir        string `yaml:"dir"`
-	TableName  string `yaml:"table"`
-	SchemaName string `yaml:"schema"`
+	Dialect    string `yaml:"dialect" envconfig:"DIALECT"`
+	DataSource string `yaml:"datasource" envconfig:"DATASOURCE"`
+	Dir        string `yaml:"dir" envconfig:"DIR"`
+	TableName  string `yaml:"table" envconfig:"TABLE"`
+	SchemaName string `yaml:"schema" envconfig:"SCHEMA"`
+}
+
+func ConfigPresent() bool {
+	_, err := os.Stat(ConfigFile)
+	return !os.IsNotExist(err)
```

Comment:
isn't this simply `return os.IsExist(err)` ?

Reply:
@hchang-clypd I think that `os.IsExist` is for error cases that signal that a file already exists: https://golang.org/pkg/os/#IsExist.

I could be reading it wrong, let me know what you gather from the documentation.

Reply:
...that's so confusing.  you read it right, i read it wrong

Reply:
definitely agree that it is confusing :grimacing: thanks for confirming with me though!